packages/atomic/src/components/commerce/atomic-commerce-interface/analytics-config.ts

```diff
@@ -7,18 +7,8 @@ export function getAnalyticsConfig(
   commerceEngineConfig: CommerceEngineConfiguration,
   enabled: boolean
 ): AnalyticsConfiguration {
-  const defaultConfiguration: AnalyticsConfiguration = {
```

Comment:
This was defining properties which do not exist on the analytics configuration for commerce, by using the `AnalyticsConfiguration` type directly.

I think this hints that I'm making a breaking change. Instead of defining a new type, and exporting it as `AnalyticsConfiguration`, I'll simply override the current analytics definition with the new one.